pkg/version/version.go

```diff
@@ -52,13 +53,16 @@ func (info Info) String() string {
 }
 
 // NewVersionCommand creates a command for displaying the version of this binary
-func NewVersionCommand(basename string) *cobra.Command {
+func NewVersionCommand(basename string, printEtcdVersion bool) *cobra.Command {
```

Comment:
would we just keep augmenting this with additional flags, e.g. `printClientFeatures bool`?